src/components/search/FacetList.jsx

```diff
@@ -76,24 +69,22 @@ const BaseFacetList = ({
 
   return (
     <div className="facet-list">
-      <OutsideClickHandler onOutsideClick={handleOutsideClick}>
```

Comment:
removing the `OutsideClickHandler` here since its no longer an installed package. `Collapsible` gets replaced here with `Dropdown` here: https://github.com/edx/frontend-app-learner-portal-enterprise/pull/44/files#diff-2cff2045e769e33d8c59d2b388691b6aR73